tests/test_multiline_variables.py

```diff
@@ -61,13 +62,13 @@ def test_multiline_variable_stdin(tmp_path):
     env['PYTHONPATH'] = os.getcwd()
     common_args = {"capture_output": True, "text": True, "env": env}
 
-    result = subprocess.run(["python3.7", script_path, "-a", "-"], **common_args, input=text, check=True)
+    result = subprocess.run(["python" + PYTHON_VERSION, script_path, "-a", "-"], **common_args, input=text, check=True)
```

Comment:
https://www.python.org/dev/peps/pep-0394 recommends to use `sys.executable`.
It is not suitable for tests in docker, but should work fine for host.
It is up to you to decide which way is better.

Reply:
The output of `sys.executable` is `/usr/bin/python3.7`. I personally don't like it and prefer `python3.7` everywhere.